static/build/miyoo/app/lang/pl.lang

```diff
@@ -0,0 +1,160 @@
+	"55": "Dodaj do ulubionych",
+	"56": "Aktualizacja",
+	"57": "Aktualizuj system?",
+	"58": "Mało miejsca na karcie SD\n%dM pozostało",
+	"59": "Numer modelu",
+	"60": "Numer seryjny",
+	"61": "Wersja oprogramowania",
+	"62": "Częstotliwość procesora",
+	"63": "%dMHz",
+	"64": "Rozmiar pamięci",
+	"65": "% dB",
+	"66": "Wykorzystana/całkowita pamięć",
+	"67": "Resetuj ustawienia",
+	"68": "Zresetować wszystkie ustawienia systemowe?\nZawartość karty SD\nnie zostanie usunięta.",
+	"69": "Odśwież romy na karcie SD",
+	"70": "Resetuj",
+	"71": "domyślny",
+	"72": "obróć o 90",
+	"73": "mapa3",
+	"74": "nieznany",
+	"75": "ŁADOWANIE",
+	"76": "Ładowanie",
+	"77": "Wyczyścić ostatnią listę?",
+	"78": "Uwaga: zawartość karty SD\nnie zostanie usunięta.",
+	"79": "Wyczyścić listę ulubionych?",
+	"80": "Uwaga: zawartość karty SD\nnie zostanie usunięta.",
+	"81": "Usunąć z listy ulubionych?",
+	"82": "Uwaga: zawartość karty SD\nnie zostanie usunięta.",
+	"83": "Skanowanie...",
+	"84": "INFORMACJE O GRZE",
+	"85": "Utwórz pamięć podręczną",
+	"86": "Wyłączenie",
+	"87": "Wyłączanie za %d sekund...",
+	"88": "WYBIERZ",
+	"89": "POWRÓT",
+	"90": "NAV",
+	"91": "MENU",
+	"92": "Wznów",
+	"93": "Zapisz",
+	"94": "Załaduj",
+	"95": "Głośność",
+	"96": "Wyjście do menu",
+	"97": "Stan %d/%d",
+	"98": "Brak stanu",
+	"99": "Brak miniatury",
+	"100": "Ładowanie nie powiodło się",
+	"101": "Zapisywanie nie powiodło się",
+	"102": "Gniazdo %d zapisane",
+	"103": "Pełny ekran",
+	"104": "WŁĄCZONY",
+	"105": "WYŁĄCZONY",
+	"106": "Zaawansowane",
+	"107": "Aplikacje",
+	"108": "Czy na pewno chcesz \nwyłączyć swoje urządzenie?",
+	"109": "Test przycisku",
+	"110": "Ponowne przypisanie przycisku",
+	"111": "WYJŚCIE",
+	"112": "ZAPISZ I WYJDŹ",
+	"113": "%d min",
+	"114": "Wyłączone",
+	"115": "Wyłącznik czasowy",
+	"116": "Usunąć grę z karty SD?",
+	"117": "Potwierdź, że chcesz usunąć\ntę grę ze swojej karty SD.",
+	"118": "Wyświetlacz",
+	"119": "Luminancja",
+	"120": "Odcień",
+	"121": "Nasycenie",
+	"122": "Kontrast",
+	"123": " \n \nUstawienia wyświetlania zapisane!\nProszę ponownie uruchomić urządzenie.\n \n ",
+	"124": "Zastosuj zmiany",
+	"125": "Tematy",
+	"126": "Rozmiar czcionki",
+	"127": "Domyślny",
+	"128": "Usuń",
+	"129": "Naprawa dźwięku",
+	"130": "Poprawki",
+	"131": "WIFI",
+	"132": " ",
+	"133": "Połącz",
+	"134": "Hotspot",
+	"135": "SSID",
+	"136": "Hasło",
+	"137": "Bluetooth",
+	"138": "Gra w sieci",
+	"139": "Zresetuj usługę Bluetooth",
+	"140": "Skanowanie",
+	"141": "Nie znaleziono urządzenia",
+	"142": "Bluetooth niegotowy",
+	"143": "Umieść romy w następującym katalogu",
+	"144": "Resetowanie Bluetooth",
+	"145": "Hotspot jest włączony",
+	"146": "Częstotliwość procesora",
+	"147": "Połączony",
+	"148": "Moje urządzenie",
+	"149": "Inne urządzenie",
+	"150": "Informacje",
+	"151": "Wprowadź SSID",
+	"152": "Wprowadź hasło",
+	"153": "Szukaj",
+	"154": "Błąd połączenia\nSprawdź swoje hasło\nLub zrestartuj router",
```

Comment:
A new entry is needed for the next version of Onion:
```
    "155": "Website",
```

Reply:
"155": "Strona internetowa",